staging/src/k8s.io/cloud-provider/controllers/nodelifecycle/node_lifecycle_controller.go

```diff
@@ -67,15 +67,13 @@ type CloudNodeLifecycleController struct {
 	nodeMonitorPeriod time.Duration
 }
 
+//logcheck:context // NewCloudNodeLifecycleControllerWithContext should be used instead of NewCloudNodeLifecycleController in code which supports contextual logging.
```

Comment:
you reference `NewCloudLifecycleControllerWithContext` but i don't see that function in this change, did i miss it?

Reply:
That's a left-over from a previous iteration when the `eventBroadcaster` was constructed in `NewCloudNodeLifecycleControllerWithContext` and thus needed a `ctx`. 

I then switched to constructing it in `Run`, which already had a `ctx`, thus avoiding the need for API changes.

Fixed, also above for NewCloudNodeController.

Going over those comments again I saw that I can avoid introducing a `NewWithContext` by using the same "delayed eventBroadcaster creation" pattern. Fixed.

Reply:
Update pushed, see https://github.com/kubernetes/kubernetes/compare/36aa8b1d528eb928accf39c6e2f244051848dd58..06281bb8fb9f243cdeb48801cc5784f03911f9a1 for the diff.